### ISSUE TITLE
[Question] How to get current displayed page(tab) in multiple pages?

### ISSUE THREAD

**Issue by refcpu** [microsoft/playwright#17433]
Hello.
I would like to get current displayed tab information in multiple tabs.
Please give me some advice.

```
const page1 = context.newPage();
const page2 = context.newPage();  // displayed tab
const page3 = context.newPage();
...
const pages = context.pages();
const currentPage = ?? // I wanna get page2
```

**Comment by GunjanSheth** [comment]
not sure if there is any specific such method

However, it does not matter if your Page is displayed on the screen or not, whatever action you ask, will be executed on that page.

Meaning even if you page1 is not the displayed tab or window, 
if you still try to find some locator and perform action, it will happen

`page1.locator("<your locator>").click()`

but if you want that your page should be displayed, before you perform some operation, there is the **_bringToFront_**() function on the page object, which will make sure that the tab / window is displayed to you.

so if by default page2 is displayed to you, but you want to perform some action on page3.
you should be doing
`page3.bringToFront()`

See if that helps

**Comment by mxschmitt** [comment]
Perfect answer by @GunjanSheth, closing by that!

TL;DR: There is no current page, all pages are considered as active in Playwright and are fully functionable.
